six/modules/c++/six.sicd/wscript

```diff
@@ -1,6 +1,6 @@
 NAME            = 'six.sicd'
 MAINTAINER      = 'adam.sylvester@mdaus.com'
-MODULE_DEPS     = 'scene nitf xml.lite six mem'
+MODULE_DEPS     = 'scene nitf xml.lite six mem str math.poly'
```

Comment:
This change isn't needed right?

Reply:
This module included things in str before this update. Looks like only math and sys list str as module dependencies.